package.json

```diff
@@ -23,7 +23,15 @@
     "merge:lcov": "cat ./coverage/*/lcov.info > ./coverage/lcov.info",
     "coverage": "npm run merge:lcov && cat ./coverage/lcov.info | coveralls",
     "heroku-postbuild": "npm run build",
-    "watch:scss": "gulp watchSCSS"
+    "watch:scss": "gulp watchSCSS",
+    "migrate:db": "cross-env NODE_ENV=development ./node_modules/.bin/migrate-mongo up",
+    "migrate:db-test": "cross-env NODE_ENV=test ./node_modules/.bin/migrate-mongo up",
+    "migrate:db-staging": "cross-env NODE_ENV=staging ./node_modules/.bin/migrate-mongo up",
+    "migrate:db-prod": "cross-env NODE_ENV=production ./node_modules/.bin/migrate-mongo up",
+    "migrate:db-down": "cross-env NODE_ENV=development ./node_modules/.bin/migrate-mongo down",
+    "migrate:db-test-down": "cross-env NODE_ENV=test ./node_modules/.bin/migrate-mongo down",
+    "migrate:db-staging-down": "cross-env NODE_ENV=staging ./node_modules/.bin/migrate-mongo down",
+    "migrate:db-prod-down": "cross-env NODE_ENV=production ./node_modules/.bin/migrate-mongo down"
```

Comment:
I feel adding `db` before the migrate scripts makes them abit too long. I think `migrate:db` can be `migrate` and something like `migrate:db-test` can be `migrate:test`. Please let me know how you feel about it.

Reply:
I like that suggestion. It makes it easy to type and remember. I'll ensure to make this changes. Thanks.